js/views/file_input_view.js

```diff
@@ -90,6 +90,33 @@
                         resolve(file);
                         return;
                     }
+                    
+                    
+                    //Fix image orientation
+                    //parse meta data
+                    loadImage.parseMetaData(blobOrFile, function(data) {
+                        //default image orientation
+                        var orientation = 0;
+                        //if exif data available, update orientation
+                        if (data.exif) {
+                            orientation = data.exif.get('Orientation');
+                        }
+                        var loadingImage = loadImage(
+                            blobOrFile,
+                            function(canvas) {
+                                //here's the base64 data result
+                                var base64data = canvas.toDataURL('image/jpeg');
```

Comment:
should `image/png` be rather used? Or a compression for jpeg? 
https://www.import.io/post/html5-canvas-todataurl-webm-vs-png-vs-jpeg/